srcpkgs/bootterm/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'bootterm'
+pkgname=bootterm
+version=0.5
+revision=1
+build_style=gnu-makefile
+# To overcome the linker getting handled compiler parameters (-Wl,-z,relro)
+make_use_env=yes
+if [ "$CROSS_BUILD" ]; then
+	make_build_args="CROSS_COMPILE=${XBPS_CROSS_TRIPLET}-"
+fi
```

Comment:
I would move this conditional block to the bottom like the other one too.

Reply:
I refrained to do so because xlint did not tell me to do it... But OK, will fix.

Reply:
done